extensions/jolie-embedding-legacy/src/main/java/jolie/embedding/jolie/JolieServiceLoader.java

```diff
@@ -58,22 +58,36 @@ public JolieServiceLoader( Expression channelDest, Interpreter currInterpreter,
 		System.arraycopy( ss, 0, newArgs, 2 + options.length, ss.length );
 		CommandLineParser commandLineParser = new CommandLineParser( newArgs, currInterpreter.getClassLoader(), false );
 
-		Interpreter.Configuration config = commandLineParser.getInterpreterConfiguration();
-		Interpreter.Configuration.create(
-			config.connectionsLimit(), config.cellId(), config.correlationAlgorithm(), config.includePaths(),
-			config.optionArgs(),
-			config.libUrls(), config.inputStream(), config.charset(), config.programFilepath(), config.arguments(),
-			config.constants(),
-			config.jolieClassLoader(), config.isProgramCompiled(), config.typeCheck(), config.tracer(),
-			config.tracerLevel(),
-			config.tracerMode(), config.check(), config.printStackTraces(), config.responseTimeout(), config.logLevel(),
-			config.programDirectory(), config.packagePaths(),
+		Interpreter.Configuration config = Interpreter.Configuration.create(
+			commandLineParser.getInterpreterConfiguration().connectionsLimit(),
```

Comment:
To shorten this out, could you put `commandLineParser.getInterpreterConfiguration()` in a temp variable (`cmdConfig`?) before these lines and then use that here?